### ISSUE TITLE
Slackit 2018 not working

### ISSUE THREAD

**Issue by linebox-chris** [pix3lot/Slackit#21]
Hello

I am trying to install this super great addin for Revit 2018. But each time I open Revit I come across the error shown below and the plugin button in the Revit ribbon is not shown. 
I downloaded the zip for 2018. Unblocked the zip then extracted the contents directly to the addin folder for Revit 2018. Am I missing something? I have only Revit 2018 installed on my PC. 
I have gone through all the past issues but none of the solutions have helped.
Any help would be greatly appreciated!

Thanks
Chris

![image](https://user-images.githubusercontent.com/43483991/45907609-9f626480-bdc6-11e8-84cc-9f04b1a2c454.png)

**Comment by pix3lot** [comment]
I'm not sure exactly what would be going on, unless it is inadvertently referencing a different Revit version's dll. I rebuilt the[ 2018 files](https://github.com/pix3lot/Slackit/blob/master/SlackIt-2018.zip) and uploaded it, can you try those?

**Comment by linebox-chris** [comment]
Thanks for the quick response. Unfortunately this did not do the trick. We are still experiencing the same error message. I tired on two separate machines. We are both using Revit 2018.3 and the free version of Slack. Could that have any affect on this?

**Comment by pix3lot** [comment]
If you right-click on the individual dlls, are those blocked? Is your Revit installed in the default location (i.e. C:\Program Files\Autodesk\...)? Does the plugin run at all or fail as soon as you click the button?

**Comment by linebox-chris** [comment]
Yes, all files are unblocked. Revit is in the default location. And the plugin does not run at all. As soon as I open Revit I get the message shown above, and once I click close, the button does not exist in the ribbon menu bar. 

![image](https://user-images.githubusercontent.com/43483991/45959080-137b5300-bfe7-11e8-913a-7e222c532f9e.png)

**Comment by pix3lot** [comment]
Apologies for the slow response I am honestly not sure why it isn't running on your setup as it should be. I tried [another rebuild](https://github.com/pix3lot/Slackit/blob/master/SlackIt-2018.zip) to see if that fixes it. If that doesn't work can you try opening the project in VS on your machine, rebuild it, and run it in debug mode? You might be able to see where it is failing.
